feathr_project/feathr/client.py

```diff
@@ -169,8 +176,23 @@ def __init__(self, config_path:str = "./feathr_config.yaml", local_workspace_dir
 
         self.secret_names = []
 
+        # initialize config helper
+        self.config_helper = FeathrConfigHelper()
+
         # initialize registry
-        self.registry = default_registry_client(self.project_name, config_path=config_path, credential=self.credential)
+        self.registry = None
```

Comment:
Shall we still wrap line [183,194] into a registry initialization function?

Reply:
> Shall we still wrap line [183,194] into a registry initialization function?

Good point, I tried to do that as well but it needs to pass around quite a few parameters like tags etc., so eventually I think making it part of the main body might be better